com.unity.ml-agents/Runtime/Actuators/ActionSpec.cs

```diff
@@ -1,15 +1,21 @@
 using System;
-using System.Collections.Generic;
 using System.Linq;
 using Unity.MLAgents.Policies;
+using UnityEngine;
 
 namespace Unity.MLAgents.Actuators
 {
     /// <summary>
     /// Defines the structure of an Action Space to be used by the Actuator system.
     /// </summary>
-    public readonly struct ActionSpec
+    [Serializable]
+    public struct ActionSpec
```

Comment:
@surfnerd Removing `readonly` here and on `BranchSizes` is OK as far as backwards compat is concerned, right?

Reply:
This is actually a breaking change.

Reply:
i had hoped, but seems like no.